### ISSUE TITLE
How do you create a wallet?

### ISSUE THREAD

**Issue by italomaia** [aeonix/aeon-legacy#141]
It would be nice to have the process documented.

**Comment by stoffu** [comment]
Just run `simplewallet`. The usage should be self-explanatory.

**Comment by italomaia** [comment]
Not really, @stoffu I think there are a lot of questions envolved. Should one run the aeond before creating the wallet? What are the generated files? How should you keep them? Had to look up a few sparse tutorials before settings things up correctly. Although, those are very simple questions docs could cover. Just my opinion.

**Comment by stoffu** [comment]
Users need to educate themselves at least a bit to understand how the CryptoNote protocol works. Without the basic understanding, it's impossible to make sense of all those software components. I've read the CryptoNote whitepaper carefully and understood how it works, so everything makes sense and the existing documentation feels sufficient to me. 

Also, Aeon isn't a product made by a company, it's an open-source project. If you think you can create a better tutorial, feel free to do so and propose changes to the codebase or website repository in the form of Pull Request. Or you can build your own tutorial site like this one: https://www.hiddenjadestone.com

> Should one run the aeond before creating the wallet? 

In practice, yes, but it's also totally fine to run simplewallet without running aeond to create a new wallet. You can receive coins at the wallet address, although you can't check the correct balance and spend funds without running aeond and fully syncing the blockchain.

> What are the generated files? How should you keep them? Had to look up a few sparse tutorials before settings things up correctly. 

simplewallet creates three files: WALLET, WALLET.keys, WALLET.address.txt. The .keys file stores your private key and requires password to open. As I said, this seems pretty straightforward and self-explanatory. And as you just did, it's not that hard to find some tutorial sites on the web. Again, if you find them too sparse and scattered, just make your own tutorial website and publish. You can't demand other people to do that for you.

**Comment by gig-tmb** [comment]
Welcome at my pool by the link and here

https://myaeon.ru/en/#wallet

You can find how create wallet offline also
